llvm/lib/Target/RISCV/RISCVVectorPeephole.cpp

```diff
@@ -498,10 +500,12 @@ bool RISCVVectorPeephole::foldVMV_V_V(MachineInstr &MI) {
                                               *Src->getParent()->getParent()));
   }
 
-  // Use a conservative tu,mu policy, RISCVInsertVSETVLI will relax it if
-  // passthru is undef.
```

Comment:
Why is this not stand for cases in the new test?

Reply:
It still stands, but the passthru isn't undef in the new test so after RISCVInsertVSETVLI a tu policy gets emitted, even though the original policy was ta